google/resource_dns_record_set_test.go

```diff
@@ -176,6 +176,40 @@ func testAccCheckDnsRecordSetExists(resourceType, resourceName string) resource.
 	}
 }
 
+func testAccCheckDnsRecordSetValid(t *testing.T) {
```

Comment:
capital T

Reply:
Thank you. Updated, and it does work now